packages/ilib-po/src/Generator.ts

```diff
@@ -72,7 +86,9 @@ class Generator {
         this.pathName = options.pathName;
         this.targetLocale = new Locale(optionsWithDefaults.targetLocale);
         this.contextInKey = optionsWithDefaults.contextInKey;
-        
+        this.datatype = optionsWithDefaults.datatype;
+        this.projectName = optionsWithDefaults.projectName ?? "default";
```

Comment:
Please move `"default"` to the `optionsWithDefaults` to keep all default values together

Reply:
Fixed